physical/s3/s3.go

```diff
@@ -72,6 +72,22 @@ func NewS3Backend(conf map[string]string, logger log.Logger) (physical.Backend,
 			}
 		}
 	}
+	s3ForceStylePathStr, ok := conf["s3_force_style_path"]
```

Comment:
Please change this and the following value to s3ForcePathStyle, which matches the actual parameter (PathStyle instead of StylePath).